airflow/models/dag.py

```diff
@@ -225,7 +226,7 @@ def __init__(
         self._description = description
         # set file location to caller source path
         self.fileloc = sys._getframe().f_back.f_code.co_filename
-        self.task_dict = dict()  # type: Dict[str, TaskInstance]
+        self.task_dict = dict()  # type: Dict[str, BaseOperator]
```

Comment:
```suggestion
        self.task_dict:  Dict[str, BaseOperator] = dict()
```
On master, we aim to Python 3+.  In the lines above - 206, we use this type of annotation.

Reply:
Sorry. I have now noticed that type annotations for variables are only supported since Python 3.6. We support Python 3.5. I cancel my comment.